test/should.js

```diff
@@ -721,6 +865,59 @@ describe('should', function() {
     }, "blah: expected 1 to have property 'length'");
   });
 
+  it('most(n) (dates)', function(){
+    var now = new Date();
+    var oneSecondBefore = new Date(now.getTime() - 1000);
+    var oneSecondAfter = new Date(now.getTime() + 1000);
+    var nowUTC = now.toUTCString();
+    var beforeUTC = oneSecondBefore.toUTCString();
+    var afterUTC = oneSecondAfter.toUTCString();
+
+    (now).should.be.at.most(now);
+    (now).should.be.at.most(oneSecondAfter);
+    (now).should.not.be.at.most(oneSecondBefore);
+
+    err(function(){
+      (now).should.be.at.most(oneSecondBefore, 'blah');
+    }, "blah: expected " + nowUTC + " to be at most " + beforeUTC);
+
+    err(function(){
+      (now).should.not.be.at.most(oneSecondAfter, 'blah');
+    }, "blah: expected " + nowUTC + " to be above " + afterUTC);
+
+    err(function(){
+      ([]).should.have.length.of.at.most(now, 'blah');
+    }, "blah: the argument to most must be a number");
+
+    err(function(){
+      ('').should.not.have.lengthOf.at.most(now, 'blah');
+    }, "blah: the argument to most must be a number");
+
+    err(function () {
+      (now).should.have.length.of.at.most(0, 'blah');
+    }, "blah: expected " + nowUTC + " to have property 'length'");
+
+    err(function () {
+      (now).should.not.have.lengthOf.at.most(0, 'blah');
+    }, "blah: expected " + nowUTC + " to have property 'length'");
+
+    err(function () {
+      (now).should.be.at.most(0, 'blah');
+    }, "blah: the argument to most must be a date");
+
+    err(function () {
+      (now).should.be.at.most(null, 'blah');
+    }, "blah: the argument to most must be a date");
+
+    err(function () {
+      (1).should.not.be.at.most(now, 'blah');
+    }, "blah: the argument to most must be a number");
+
+    err(function () {
+      (now).should.not.be.at.most(undefined);
+    }, "the argument to most must be a date");
+  });
+
```

Comment:
Don't need parenthesis around `<var name>` whenever it's `(<var name>).should`; still need parenthesis for the values plugged directly, like `(0).should`